### PR TITLE
Intro Dialog Fixes

### LINKED ISSUE #2795: Modal dialogs are not centered correctly
image

image

### PR DESCRIPTION
**Pull request by placegraphichere**
- Next/prev buttons removed, just goes to the next. Back button returns
to the zero slide: #2749
- [edit] now fixes  #2795

### CODE REVIEW

**Comment by designedbyscience** [comment]
Are we no longer centering these dialogs on the canvas?

**Comment by placegraphichere** [comment]
@designedbyscience see #2795, I describe the issue with centering there. Basically we need additional JS to center, not just CSS due to the possible column expand/collapse. I've probably got the code we need in the branch about positioning the export dialog from a few weeks ago

**Comment by designedbyscience** [comment]
There is already a CSS class on div.main with one, two or no panels, to adjust the centering

**Comment by placegraphichere** [comment]
neato, I'll check it out and update this

**Comment by placegraphichere** [comment]
Now should fully address #2795. 

When width of app is too small based on 2 columns:
screen shot 2015-10-12 at 4 37 02 pm
screen shot 2015-10-12 at 4 30 22 pm

---

When different columns are open:
screen shot 2015-10-12 at 3 51 02 pm
screen shot 2015-10-12 at 3 50 50 pm
screen shot 2015-10-12 at 3 50 37 pm
screen shot 2015-10-12 at 3 50 16 pm
screen shot 2015-10-12 at 3 49 28 pm
screen shot 2015-10-12 at 3 49 16 pm

File: src/style/help/first-launch.less [hunk 1]
```diff
@@ -32,6 +32,36 @@
     height:66rem;
     border-radius: 0.8rem;
     overflow: hidden;
+    margin-left: 0;
+}
+.main__no-panel .first-launch__dialog, .main__no-panel .keyboard-shortcut__dialog{
```
Comment (designedbyscience) [inline src/style/help/first-launch.less]: Spaces before braces

**Comment by iwehrman** [comment]
This does not fully address #2795. It seems like single-column mode is not taken into account, at least not consistently:
image

I think we should just get rid of the back arrow. The asymmetry feels weird, and the new behavior doesn't seem any more intuitive me than the old behavior. Could we just have an icon for the intro slide instead of the "home" button?

**Comment by placegraphichere** [comment]
this is in a better state now. assigning to @iwehrman since he's still here today

**Comment by iwehrman** [comment]
I agree this is in a better state. Merging away!

**Pull request merged**